app/layout.tsx

```diff
@@ -34,6 +33,7 @@ export default function Layout({ children }: { children: ReactNode }) {
4uA0jDto5u+HI4mS3A7Yh/YClZ1YzeLs3Bx67/njWDo3sB0QMIeJidU01LwSaZxXetlbIhEqcw6gCVGT8zl0GgbSOL/QTtesVdzAAtQ3zuSJ2TgNQwyTiESOML0sB2cyiWECkcgBpidWEYVEMZNIhDkii5jegYooJIsxfjZYeGoJmlg1WdIuEoXoa//w6Kvex9T8/91bv9M+B1Rmcgw/Q3tQSdvXWCj52kk1kNLJt69e6Z2PCoyC6pxNhjVhLAMt78kbhbK+tsmVEuCIk6NOAQPloL1CJidWibzp2KAGkBSNYu/l7QNEIkv0emYbjEJ0KmDeVGzYPqVeiofGJiZBOmeDIk0Y85J3LDSqnE5io5e3US5BIguYHnsUi0Kjy+no5W0WSGQYOt1uWCQoQ94oVKT11rCUDxQDEhnG+I7VnFGoaCGDRCfhQXUgkUGyTrerSv4oVLyQYVr4ugKJDDHsdLuy5I1CZQsZJDylgKAakMgQppowpjl/9652LYsqEYVSQBQZqgGJDGCyCWPC6fduqFON0eeXUgpZpekJenlXBxIZwMaLMM9YyFQKiQPDKrELiSpCUYhehCY5MzeXKwrROMhUCokiQ2l2TjbWVtcDffIi+MHwygRqgXVuIV9FzmQKSSkh1tWVA5GoAqa79yhugZWn+YjpfUpEsvfIJyE25odEFaAX+ztXrxj7/6iYcHb+19r1LKj8TVHLJBJaEb9+vK1dE85OsilvF7tby3H+zh11qnFRvdraLP1/0G5VKiTkGQcdPqbRUOfm540VNWivUl6BwTEOJdqBROWhF99Z5f4FeGbuAzV2eUr9r+JavZOCGp+86ZpNUV2A7eGBQysaYtrybXqc54DNZEyECh0QQYBnNj1FYQGIIdAJ3x1EIiAG05PWLmisrR5K9BQvJeCbAPc39UJnT6LG2mr5+iwABqBULsCiAlW1j022YgUi8MYPj8wun3JEL/ikJdrBSwj4gBbxmu6O5AhNIqR0wAsB72fS0jlU6IBzaCwUYlVOpWoJiETAG7QK3vSxMw7ZSL7UoUSNtdWnvBAVACdQGhdgRS7hMOj0r1hASgecQPuhAi0mJBy60i8RUjpgHRIogg6sAyVCJAJWiUSgXR7+9DgmEVcb9rSHAGCASAQiVtL/yFrFvaJdAaAi1E0ooib6xzK2LImQ0gFjvN7eVt9++FvjzS09c8yRrJ2tFInua1cBKADNAdFhYqb7kwtgIz0eUlkS0Sd0my1ajIpjpkFhEnmolViI7a9yoA13NImYZaXUZ9pVADIgcV5tbqrvHz0KdglPAXJLtAKJQBbJFu43T7q9rqmvtx+H2BehLFu0k7X/sZkS0ScipcsPDZ7/c/ujWNMXcMRy1s8iqzqXkPkAcBwIVCu0VE6NkCjzAeAICFQrMlM5NUwifsCGdgP0gEC1Y2BmNlAiZuAD6wwEqiUDXRgqUWNtdRlr6Y4DgWpJp3+CNc1QiZiBBtYNCFRbhjqQR6K2dqWGQKDasjvqNMmREqHAAIFqztKobz9vQ/uR/1GsQKBas5dnqieXRBzOatfEBALVnvawgkJCkaNVahWNIFDt2ctbD8gtEZe7axGNIBCgNC5PFFIlTg+PPhpBIMDkfq0Xkij2aASBANMZtE4uizLHTUYZjSAQYGgstFjkh1FYohijEQQCKXJV5NKUPfh4QbsSKBAIpMhdkUtTSiKeNwp+FQMEAn0sFo1CqkIkUqFHIwgE+tjioUphSkvE1Ys/aTcCAAKBDAoVE9JUiUSKK3Xe9huRDGUeA4FAH51RK7WHUUkizh+9pXVFz7eBQCCDwiXtfqpGIhKJVrk+1G44gPo7541GEAgMoFQxIU1liZgFX2kdiZE0FBwE3YdAIIONssWENCf29/e1i2XoNluzSqkvff2m3r56RY1Pz6jxmWk1NjXVu0bRh6JV4McaAjvQH/2ZIst7BmFMInUgEqV272s3AJDHL3koUhlT6VyCt7QOgAI8NCWQMi0RD9BmtRsAyGHXdEXZdCRKlgT9XrsBgAxmq1bj+jEukToQaanuHYKASG7z4d5GsSIRM1vH5iZALLQqwUoPRWsSpcZHKDQA32xVXZUwDJuRSHHotPbkAcjBno1xUBqrEqmjnbAoNAAfkEDXTUyoDsO6ROqo0BDdWexAPIs2Cgn9GF2xMIpus7WJc2CBI26aWBeXByeRKMV1HuQBYJOOK4GUa4l4cAeRgE1IIKd73FxHIogEbOJcIOVDIgWRgB28CKR8SaQgEjCLN4GUT4kURAJm8CqQ8i2RgkigGt4FUhIkUsdFwspvkJebEgRSridb89Bttqi+Py/qSQFpOJtIzYOISJSG/7pgrR3IgtbC/VySQEqiROpord1NbKMAKba4O4/1tXBFEZfOpek2WzN8BPol7SaoE9QcdMHmdoYqiJZIHYg0oZRaRiuu2nLb1o5UU4iXKKHbbFGK97F2A8RKspmudKN5VwQjkUJ6VydEp2/9BCWROkrv2iiDRwlFnyXp6Vs/wUmUwL2/aax0QbsJQmSDo4/Vrdw2CFYihaJDLAQZfdIELVFCt9m6zjJhrBQWD7kPQnDRJ00UEiVwBW8RKZ54dlkeY03lfRKVROpApEk+SxaFB3lQ6tbmFSnREJ1ECZzi0S/rmnYT+KBj4mhHiUQrUQLL1EarLm90uHAQ9LhnGNFLlNBtthZ4vASZ3BC9PAm1kSgBaZ51aiNPQu0kSuAlRIsoQBhhj1PmdoxjnlHUVqIEnrBd5CMIMc9UDFplsCxtk5xrai9RGk71FhCdhrLLi4DbdUrZhgGJMuDoNMtvWFJ0kK6tcNQRvzXBNZBoBH1CXa/Raogk4qzHsrLAFpCoIJzyJULFVC6naLOevEnsZSAVSFQBjlIk0wy/D6lsTpFmE9JUBxIZhkvn9DbJYk0KqPpRp5ydlDSbdSxF2wISOYLlmmDBJliuSf7qVUQjQRIhNvnjneQNFTTLKKX+D7rIQEhQx22LAAAAAElFTkSuQmCC" />
```

Comment:
might be nice to store the logo as a variable in a "resources" file and import so it doesn't bulk the `app/layout.tsx` here.